idaes/models_extra/gas_solid_contactors/properties/oxygen_iron_OC_oxidation/solid_phase_thermo.py

```diff
@@ -413,49 +486,54 @@ def build(self):
 
     def _make_state_vars(self):
         """List the necessary state variable objects."""
+
+        # create units object to get default units from the param block
+        units_meta = self._params.get_metadata().derived_units
+
         self.flow_mass = Var(
             initialize=1.0,
             domain=Reals,
-            doc="Component mass flowrate [kg/s]",
-            units=pyunits.kg / pyunits.s,
+            doc="Component mass flowrate",
+            units=units_meta["mass"] / units_meta["time"],
         )
         self.particle_porosity = Var(
             domain=Reals,
             initialize=0.27,
             doc="Porosity of oxygen carrier [-]",
-            units=pyunits.m**3 / pyunits.m**3,
+            units=units_meta["length"] ** 3 / units_meta["length"] ** 3,
```

Comment:
Due to the way `pint` works, you might as well just make this `pyunits.dimensionless`.